identitynow.sgnl.yaml

```diff
@@ -0,0 +1,239 @@
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: name
+        externalId: name
+        type: String
+      - name: created
+        externalId: created
+        type: DateTime
+      - name: modified
+        externalId: modified
+        type: DateTime
+      - name: attribute
+        externalId: attribute
+        type: String
+      - name: value
+        externalId: value
+        type: String
+      - name: sourceSchemaObjectType
+        externalId: sourceSchemaObjectType
+        type: String
+      - name: privileged
+        externalId: privileged
+        type: Bool
+      - name: cloudGoverned
+        externalId: cloudGoverned
+        type: Bool
+      - name: description
+        externalId: description
+        type: String
+      - name: requestable
+        externalId: requestable
+        type: Bool
+      - name: segments
+        externalId: segments
+        type: String
+        list: true
+  AccountEntitlement:
+    displayName: AccountEntitlement
+    # This externalId does not need to match the IDN's API unlike the rest of the entities. AccountEntitlement is an entity solely used
+    # to create the relationship between Entitlement and Account.
+    externalId: accountEntitlements
+    description: AccountEntitlement entity in IdentityNow
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 250
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: accountId
+        externalId: accountId
+        type: String
+        indexed: true
+      - name: entitlementId
+        externalId: entitlementId
+        type: String
+        indexed: true
+  Identity:
+    displayName: Identity
+    externalId: identities
+    description: Identity entity in IdentityNow
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 250
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: name
+        externalId: name
+        type: String
+      - name: created
+        externalId: created
+        type: DateTime
+      - name: modified
+        externalId: modified
+        type: DateTime
+      - name: alias
+        externalId: alias
+        type: String
+      - name: emailAddress
+        externalId: emailAddress
+        type: String
+      - name: processingState
+        externalId: processingState
+        type: String
+      - name: identityStatus
+        externalId: identityStatus
+        type: String
+      - name: isManager
+        externalId: isManager
+        type: Bool
+      - name: lastRefresh
+        externalId: lastRefresh
+        type: DateTime
+
+relationships:
```

Comment:
Suggest creating the relationship between Accounts and an Identity too, via the identityId attribute on accounts to the id attribute on identities.

Reply:
Added the `AccountIdentity` relationship between `Account -> Identity`. Open to other names.